NEWS

```diff
@@ -55,6 +55,43 @@ NEWS - user visible changes				-*- outline -*-
    If LS_NULLS is active and invalid data (bad encoded or missing encoding)
    is found io status 71 is returned.
 
+** new -fformat dialect option, and extended SOURCE FORMAT directives, with
```

Comment:
this is currently duplicate (merge-conflict, I guess), it belongs below "change to options", not "to change to existing programs"

Reply:
Ouch! Thanks for catching that!